hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/connector/StreamEventJournalP.java

```diff
@@ -291,15 +299,6 @@ private void tryGetNextResultSet() {
                 // This exception can be safely ignored - it means the instance was shutting down,
                 // so we shouldn't unnecessarily throw an exception here.
                 return null;
-            } else if (ex instanceof StaleSequenceException) {
```

Comment:
You can still get the exception if the operation is parked on the member side (and may have read some items) after which the partition gets lost so you might want to keep this block.

Reply:
we use minFetchSize=1, so it should not get parked after after it read some items